pkgs/development/libraries/totem-pl-parser/default.nix

```diff
@@ -25,11 +25,16 @@ stdenv.mkDerivation rec {
     "-Dintrospection=false"
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    # Upstream MR: https://gitlab.gnome.org/GNOME/totem-pl-parser/-/merge_requests/46
+    sed -i "s/,--version-script,.*$/'/" plparse/meson.build
+  '';
```

Comment:
I would prefer to fetch the patch and make this unconditional so that it is not forgotten to be removed again

Reply:
Done.